### PR TITLE
frontend: Refresh look

### PR DESCRIPTION
**Pull request by juliaogris**
Refresh look with Jas' generous design donation:
https://www.figma.com/file/4Wf4Mbq8I2Yxh2bptOnSE9/evy?node-id=11%3A240&t=9o6LjggxGbWl61xR-0

### CODE REVIEW

**Comment by camh-** [comment]
I seem to get more scroll bars than you do: 
image


image

Not sure why this is

**Comment by juliaogris** [comment]
> I seem to get more scroll bars than you do: image
> 
> image
> 
> Not sure why this is

man that's annoying. Not really sure how to debug this

> I seem to get more scroll bars than you do: image
> 
> image
> 
> Not sure why this is

Ok, thanks for that. I will have a play around... not really sure what to do. Maybe I could debug it on your machine / with you next time we hang?

**Comment by camh-** [comment]
> not really sure what to do

Let's not worry about it for now - if it is working fine for you, this can be merged and we can figure thus out later. Probably I've managed to somehow make my system non-standard (again).

**Review by camh- — APPROVED** [review]
:cucumber: Visually, looks good to me. Can't say much about  the CSS/JS :moneybag:

File: frontend/index.html [hunk 1]
```diff
@@ -36,24 +46,70 @@
...
+            <label>input&gt;</label>
+            <textarea id="read" rows="1"></textarea>
+          </div>
+          <div class="input slider hidden">
+            <input type="range" id="sliderx" min="0" max="100" />
+            <input type="range" id="slidery" min="0" max="100" />
+          </div>
+          <!-- prettier-ignore -->
+          <div class="console" id="console">x: 12
+x: 12
```
Comment (camh-) [inline frontend/index.html:65]: Why is the output window pre-loaded with this? Shouldn't it be empty until "run" is pressed?

Regardless, if you do want it, "x: 12" is here twice but when you run the welcome program, it only prints it once.

Comment (juliaogris) [reply frontend/index.html]: fixed, thanks for the comment.

**Comment by juliaogris** [comment]
> > not really sure what to do
> 
> Let's not worry about it for now - if it is working fine for you, this can be merged and we can figure thus out later. Probably I've managed to somehow make my system non-standard (again).

@camh- I realised I got a similar behaviour to you on Firefox, at least for the double scrollbar. I've tried to fix it....

**Comment by camh-** [comment]
Looks better now. No nested or unnecessary scroll bars

**Pull request merged**